aws-py-static-website/README.md

```diff
@@ -0,0 +1,138 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Secure Static Website Using Amazon S3, CloudFront, Route53, and Certificate Manager
+
+This example serves a static website using TypeScript and AWS.
+
+This sample uses the following AWS products:
+
+- [Amazon S3](https://aws.amazon.com/s3/) is used to store the website's contents.
+- [Amazon CloudFront](https://aws.amazon.com/cloudfront/) is the CDN serving content.
+- [Amazon Route53](https://aws.amazon.com/route53/) is used to set up the DNS for the website.
+- [Amazon Certificate Manager](https://aws.amazon.com/certificate-manager/) is used for securing things via HTTPS.
+
+## Getting Started
+
+Configure the Pulumi program. There are several configuration settings that need to be
+set:
+
+- `targetDomain` - The domain to serve the website at (e.g. www.example.com). It is assumed that
+  the parent domain (example.com) is a Route53 Hosted Zone in the AWS account you are running the
+  Pulumi program in.
+- `pathToWebsiteContents` - Directory of the website's contents. e.g. the `./www` folder.
+
+## Deploying and running the program
+
+Note: some values in this example will be different from run to run.  These values are indicated
+with `***`.
+
+1.  Create a new stack:
+
+    ```bash
+    $ pulumi stack init website-testing
+    ```
+
+1.  Set the AWS region:
+
+    ```
```

Comment:
Nit: Since you put `bash` as the language type for the previous code block, we should be consistent here and the other snippets. (Though it won't actually _do_ anything, but in a perfect world it would lead to better syntax highlighting when rendered on GitHub.)